### DISCUSSION TITLE
稼働スケジュール共有

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#6] [General]
#稼働できない時間

### Nori
- 12月20日 10:00~12:00 kid swimming
- 12月20日 17:30～ 忘年会
- 12月21日 19:00~ Kid対応
- 12月22日 15:30~19:00 バリスタトレーニング
- その他　18:30-19:30 不定期でKid寝かしつけ対応

**Comment by tkg-geek** [comment]
#稼働できない時間

**たけうってぃー**
- 12月21日（土）11:00-17:00 福岡〜東京移動
- 12月23日（月）18:00-22:00 バドミントン🏸

**Comment by mikiIDE** [comment]
#稼働できない時間

### いでち
- 12月20日 14:00～20:00　講義+実家への帰省
- 12月21日 ～12:00　実家の愛犬の病院

それ以外は基本的に調整可能です！

**Comment by NoRio-kd** [comment]
すみません、明日のスケジュール共有もれがあったので更新しました。

【更新後】
12月20日 10:00~12:00 kid swimming
